fastlane/spec/cli_tools_distributor_spec.rb

```diff
@@ -50,6 +50,8 @@
       expect do
         Fastlane::CLIToolsDistributor.take_off
       end.to raise_error(SystemExit)
+
+      `git checkout fastlane/README.md`
```

Comment:
That seems like a hack, can we just not generate the file in the first place instead?